packages/palette/src/elements/Skeleton/Skeleton.tsx

```diff
@@ -75,3 +76,31 @@ export const Skeleton: React.FC<SkeletonProps> = ({ children, ...rest }) => {
     </Box>
   )
 }
+
+export const SkeletonImage: React.FC<SkeletonProps & { blurhash?: string }> = ({
+  children,
+  blurhash,
+  ...rest
+}) => {
+  return (
+    <Box position="relative" {...rest}>
+      {!!blurhash && (
+        <BlurhashCanvas
+          hash={blurhash}
+          style={{
+            position: "absolute",
+            top: 0,
+            bottom: 0,
+            left: 0,
+            right: 0,
+            width: "100%",
+            height: "100%",
+            zIndex: 99,
```

Comment:
@dzucconi @MrSltun @anandaroop I added this `zIndex: 99` as that will hopefully cause it to show up properly in Force (🤞 ).

There is a noticeable delay from storybooks (likely that 500ms timeout), which at least for demo purposes is actually probably not bad.

Without `zIndex` being specified at all, there seems to be some kind of 'double-flash' that shows up in Storybooks.

![Jan-18-2024 18-25-25](https://github.com/artsy/palette/assets/1457859/03ee9d00-cffa-46e4-bf6b-93a2cb0c6da5)